src/modules/FileLocksmith/FileLocksmithUI/Views/MainPage.xaml

```diff
@@ -139,19 +129,18 @@
                                             </ToolTipService.ToolTip>
                                         </FontIcon>
                                         <Button
-                                            MinWidth="132"
+                                            MinWidth="128"
                                             Command="{Binding Path=DataContext.EndTaskCommand, ElementName=ProcessesListView}"
                                             CommandParameter="{Binding}">
                                             <StackPanel
                                                 Orientation="Horizontal"
-                                                Spacing="6">
+                                                Spacing="8">
                                                 <FontIcon
-                                                    FontFamily="{ThemeResource SymbolThemeFontFamily}"
-                                                    FontSize="14"
+                                                    FontSize="16"
                                                     Glyph="&#xf140;" />
                                                 <TextBlock
                                                     x:Uid="EndTask"
-                                                    FontSize="14" />
+                                                    FontSize="16" />
```

Comment:
Please revert, as per design spec.

See Task Manager:

<img width="277" alt="image" src="https://github.com/microsoft/PowerToys/assets/9866362/6b1d9055-1f23-4303-8d4a-9cbe501d34ca">

Reply:
looks like 14 is default :)